Apps/WebClient/src/ClientApp/src/components/dependentCard.vue

```diff
@@ -113,6 +130,21 @@ export default class DependentCardComponent extends Vue {
             });
     }
 
+    private deleteDependent(): void {
+        if (confirm(this.message)) {
```

Comment:
per mandads mockup we should be using a custom dialog instead of the browser one

Reply:
done!